From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Tue, 1 Nov 2011 15:23:06 +0100
Subject: [PATCH] use packages gtkbuiltincache.h and don't generate it

This way host gtk-update-icon-cache is not needed.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 gtk/Makefile.am |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/gtk/Makefile.am b/gtk/Makefile.am
index 267c499..2ce0bd4 100644
--- a/gtk/Makefile.am
+++ b/gtk/Makefile.am
@@ -1304,11 +1304,11 @@ gtk_update_icon_cache_program =							\
 	   ./gtk-update-icon-cache
 endif
 
-gtkbuiltincache.h: @REBUILD@ stamp-icons
-	$(MAKE) $(AM_MAKEFLAGS) gtk-update-icon-cache$(EXEEXT) $(GTK_UPDATE_ICON_CACHE_MANIFEST)
-	$(gtk_update_icon_cache_program) --force --ignore-theme-index		\
-	   --source builtin_icons stock-icons > gtkbuiltincache.h.tmp &&        \
-	mv gtkbuiltincache.h.tmp gtkbuiltincache.h
+#gtkbuiltincache.h: @REBUILD@ stamp-icons
+#	$(MAKE) $(AM_MAKEFLAGS) gtk-update-icon-cache$(EXEEXT) $(GTK_UPDATE_ICON_CACHE_MANIFEST)
+#	$(gtk_update_icon_cache_program) --force --ignore-theme-index		\
+#	   --source builtin_icons stock-icons > gtkbuiltincache.h.tmp &&        \
+#	mv gtkbuiltincache.h.tmp gtkbuiltincache.h
 
 EXTRA_DIST +=                   \
 	$(STOCK_ICONS)		\
